docker/Dockerfile

```
@@ -1,13 +1,14 @@
 FROM php:8.1-fpm
 
-COPY ./root /root
+COPY ./docker/root /root
```

```suggestion
COPY ./root /root
```
Не вижу причин менять эту строку

---

Причину описал нельзя будет скопировать  COPY /../.env  .env (такое не поддерживается)